### DISCUSSION TITLE
Add ability to select sides of individual tiles rather than just corners (like RCT3)

### DISCUSSION THREAD

**Discussion by tellovishous** [OpenRCT2/OpenRCT2#17008] [Feature requests] [answered]
Had a feature idea for a "backport" from RCT3:
When selecting individual land tiles to adjust, you can either adjust the corners or the tile as a whole. But RCT3 made it so you could also select the whole side of a tile when putting the cursor in the middle of the edge. Would be nice if we could add this ability to OpenRCT2 as well, since you could make & adjust sloped tiles in one click instead of two.

**Comment by Broxzier** [comment, marked as answer]
Assuming this is about the land tool, try holding down the Ctrl key, and it'll let you raise or lower two corners simultaneously.

**Reply by tellovishous** [reply]
Ahh, I was actually not aware of this function, great to know! Nonetheless, I think it'd still be nice to have the RCT3 functionality for tile sides without having to using a hotkey.

**Reply by Broxzier** [reply]
Changing that would mean that players have to be extra precise with where they hover if they do want to raise just one corner. I don't think that will be a good change. In RCT3 the tiles are much larger on your screen, so it is not a problem there.
